front-end/src/components/Register.js

```diff
@@ -0,0 +1,182 @@
+import React, { useState } from "react";
+import { BrowserRouter as Router , Route, Link } from "react-router-dom"; 
+
+import axios from "axios";
+const Register = (props) => {
```

Comment:
move the `axios` import to line 3 and add an empty line in  between it and `Register`